src/main/java/it/pn/frontend/e2e/pages/radd/AvvisiAvvenutaRicezionePage.java

```diff
@@ -0,0 +1,151 @@
+
+    @FindBy(xpath = "//button[contains(text(),'Ho finito')]")
+    WebElement hoFinitoButton;
+
+    @FindBy(xpath = "//button[contains(text(),'Indietro')]")
+    WebElement indietroButton;
+
+    public AvvisiAvvenutaRicezionePage(WebDriver driver) {
+        super(driver);
+    }
+
+    public void waitLoadAvvenutaRicezione() {
+        try {
+            By titlePageBy = By.xpath("//h4[contains(@class,'MuiTypography-root MuiTypography-h4 ')]");
+            this.getWebDriverWait(30).until(ExpectedConditions.visibilityOfElementLocated(titlePageBy));
+            logger.info("La pagina Avvisi di avvenuta ricezione caricata");
+        }catch (TimeoutException e){
+            logger.error("La pagina Avvisi di avvenuta ricezione NON caricata con errore: "+e.getMessage());
+            Assert.fail("La pagina Avvisi di avvenuta ricezione NON caricata con errore: "+e.getMessage());
+        }
+    }
+
+    public boolean checkPersonaFisica() {
+        return this.personaFiscaRadioButton.isSelected();
+    }
+
+    public void inserimentoCFDDestinatario(String cfAAR) {
+        this.codiceFiscaleInput.sendKeys(cfAAR);
+    }
+
+    public void inserimentoCFDelegato(String cfDAAV) {
+        this.codiceFiscaleDelegatoInput.sendKeys(cfDAAV);
+    }
+
+    public void waitLoadCaricamentoDocumenti() {
+        try {
+            By titlePage = By.xpath("//h6[contains(@class,'MuiTypography-root MuiTypography-h6')]");
+            this.getWebDriverWait(30).until(ExpectedConditions.visibilityOfElementLocated(titlePage));
+            logger.info("Caricamento Documenti section caricata correttamente");
+        }catch (TimeoutException e){
+            logger.error("Caricamento Documenti section non caricata correttamente con errore: "+ e.getMessage());
+            Assert.fail("Caricamento Documenti section non caricata correttamente con errore: "+ e.getMessage());
+        }
+    }
+
+    public void uploadDocumento(String path) {
+
+        if (this.uploadBox.isDisplayed()){
+            this.uploadBox.sendKeys(path);
+        }else {
+            this.js().executeScript("arguments[0].scrollIntoView(true)",this.uploadBox);
+            this.uploadBox.sendKeys(path);
+        }
+
+    }
+
+
+    public void clickContinuaButton() {
+        this.continuaButton.click();
+    }
+
+    public void waitLoadAvvenutaRicezioneUltimoStep() {
+        try {
+            By terzoStepARBy = By.xpath("bbb");
+            getWebDriverWait(30).until(ExpectedConditions.visibilityOfElementLocated(terzoStepARBy));
+            logger.info("La section avvisi avvenuta ricezione si visualizza correttamente");
+        }catch (TimeoutException e){
+            logger.error("La section avvisi avvenuta ricezione si visualizza correttamente con errore:"+e.getMessage());
+            Assert.fail("La section avvisi avvenuta ricezione si visualizza correttamente con errore:"+e.getMessage());
+        }
+    }
+
+    private void closeTab(String url) {
```

Comment:
@delfinoroNTTDATA non è chiaro cosa si intende fare con questo metodo? serve chiudere tutti i tab aperti dopo i click sulla timeline?

Reply:
è sbagliato il metodo, in teoria si dovrebbera fare il download dei file aperti

Reply:
@delfinoroNTTDATA quindi va aggiustato?

Reply:
lo fatto